recipes/libmo_unpack/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "3.0" %}
+
+package:
+    name: libmo_unpack
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/SciTools/libmo_unpack.git
+    git_rev: v{{ version }}
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - check
+        - automake
+        - libtool
+        - pkg-config
+    run:
+        - libgcc
+
+test:
+    commands:
+        - exit $(test -f ${PREFIX}/lib/libmo_unpack.a)  # [not win]
+
+about:
+    home: https://github.com/SciTools/libmo_unpack
+    license: BSD 3-Clause
+    summary: Met Office PP/FieldsFile compression schemes
+
+extra:
+    recipe-maintainers:
+        - pelson
+        - rhattersley
```

Comment:
@rhattersley I add you as a maintainer here and on the dependencies. Let me know if that is OK or if you want me to remove it from some (or all) the packages.